player/stream-test/js/script.js

```diff
@@ -159,7 +159,15 @@ function handleKeyPress(keyEvent) {
   }
 }
 
-function setupPlayer(manifestType, manifestUrl, drm = 'none', licenceUrl = '', autoplay) {
+function setupPlayer(manifestType, manifestUrl, drm, licenceUrl, autoplay) {
+  if (!drm) {
```

Comment:
As per the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), the behavior of default parameters is to only use the default value if `undefined` is passed.

Please fix that everywhere.